Add my JSON api

Chew opened the pull request:
it's useful and it works. perfect for developers!

API has been approved by md678685. Code works by grabbing the JSON file from GitHub on server boot, does not touch the site, very friendly.

Example:
`GET https://api.chew.pro/spigotdrama`
Response:
```json
{
  "response":"electroniccat came out against drama, bStats rages",
  "permalink":"https://drama.essentialsx.net/eyJzZW50ZW5jZSI6MzQsInBlb3BsZSI6WzRdLCJzaXRlcyI6WzE4XSwiZHJhbWEiOlsyXX0="
}
```

astei commented:
I have a better idea on how to implement this

Chew committed:
Add my JSON API

mdcfe commented on src/index.js:
```diff
@@ -79,6 +79,8 @@ function renderDrama(message, share, sharePath, teaser) {
             <br />
             <i>Now available in <a href="https://twitter.com/SpigotDrama">Twitter</a> form!</i>
             <br />
+            <a href="https://api.chew.pro/spigotdrama">JSON API</a>
```
```suggestion
            <a href="https://api.chew.pro/spigotdrama">Chew's JSON API (unofficial)</a>
```

Chew committed:
Commit suggestion

https://github.com/md678685/spigot-drama-generator/pull/12#discussion_r515531849

Co-authored-by: MD <1917406+md678685@users.noreply.github.com>
